crds/virtualmachineipaddresslease.yaml

```diff
@@ -106,13 +99,17 @@ spec:
 
                     * Bound - the IP address lease is bound to IP address.
                     * Released - the IP address lease is available for binding.
+                observedGeneration:
+                  type: integer
+                  description: |
+                    The generation last processed by the controller.
           type: object
       served: true
       storage: true
       subresources:
-        status: { }
+        status: {}
       additionalPrinterColumns:
-        - jsonPath: .spec.ipAddressRef
+        - jsonPath: .spec.virtualMachineIPAddressRef
           name: IpAddress
           type: string
```

Comment:
Something wrong here. virtualMachineIPAddressRef is an object, not a string. What we want to expose here? A string with Ip address or a string with name/namespace?

Reply:
We want expose a string with name/namespace. After renaming "claimRef" -> "virtualMachineIPAddressRef" and respectively "name: Claim" -> "name: IpAddress"

Rename "name: IpAddress" to "name: VirtualMachineIPAddress"